tests/test_autoregressive_generator.py

```diff
@@ -0,0 +1,54 @@
+import pytest
```

Comment:
Can you add the Meta Copyright header to any file added to the PR?
You can copy and paste it from other files:

```
# Copyright (c) Meta Platforms, Inc. and affiliates.
# All rights reserved.
#
# This source code is licensed under the license found in the
# LICENSE file in the root directory of this source tree.
#

```